src/board/listener.ts

```diff
@@ -109,6 +112,16 @@ export const createMessageListener = (board: BoardUI) => (e: MessageEvent) => {
         board.writeSerial(data.data);
         break;
       }
+      case "radio_input": {
+        if (
+          typeof data.data !== "string" &&
+          !(data.data instanceof Uint8Array)
+        ) {
+          throw new Error("Invalid radio_input data field.");
+        }
+        board.radio.receive(data.data);
```

Comment:
This should prepend "\x01\x00\x01" in the string case or not accept strings.
Clients kinda have to cope with bytes anyway so tempted to just deal with bytes in the sim.